{"uuid": "bfe8687e-3c8e-4050-8d27-dd898c59df5a", "children": ["366747ff-dacf-4671-857c-6cca3b395389", "10e4a29d-9f33-4ede-844d-36540eafeaa5", "393406cb-f5ef-4774-a5f3-196035315ea0"], "befores": [{"name": "init_driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: Process unexpectedly closed with status 0\n\n", "trace": "  File \"f:\\practice codes\\selenium_practice\\venv\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"f:\\practice codes\\selenium_practice\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"f:\\practice codes\\selenium_practice\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"F:\\GitHub\\Page_Object_Model-Python_Selenium_with_Pytest\\Facebook_automation\\Tests\\conftest.py\", line 23, in init_driver\n    web_driver = webdriver.Firefox(executable_path=TestData.FireFox_executable_path)\n  File \"f:\\practice codes\\selenium_practice\\venv\\lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 170, in __init__\n    RemoteWebDriver.__init__(\n  File \"f:\\practice codes\\selenium_practice\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"f:\\practice codes\\selenium_practice\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"f:\\practice codes\\selenium_practice\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"f:\\practice codes\\selenium_practice\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1629226332018, "stop": 1629226337358}], "start": 1629226332018, "stop": 1629226337605}